transmogrifier/sources/xml/datacite.py

```diff
@@ -339,6 +238,189 @@ def _get_contributors(cls, source_record: Tag) -> Iterator[timdex.Contributor]:
                     kind=contributor.get("contributorType") or "Not specified",
                 )
 
+    @classmethod
+    def get_dates(
```

Comment:
Commenting high up here, but I continue to have the feeling the dates logic and handling as a whole could be refactored some.   But, I think it could wait.  It feels like there are tethers to the `validate_date()` which consistently complicate the logic, and depending on how that lands, probably isn't the right time to tackel.

FWIW, I _do_ think this is a nice port into the field method approach, which is the goal here.

Reply:
Agreed that this is another issue better handled globally and later on